scheduler.py

```diff
@@ -44,9 +44,24 @@ def convert_to_timeform(time):
     for i in range(((24*tasks[1])-int(td.hour))-1):
         raw_time = td-tdmin+datetime.timedelta(hours=i+1)
         schedule[convert_to_timeform(raw_time)] = [np.nan]
-    def find_time():
+    def find_time(n):
         """find first instance of window of empty values in dict"""
-        ...
+        j = 0
+        index = 0
+        for i in schedule.keys():
+            
+            print(schedule[i])
+            if (np.isnan(schedule[i])):
+                j += 1
+            else:
+                j = 0
+            if (j >= n):
+                for k in range(index-n, index+1):
+                    key = list(schedule.keys())[k]
+                    schedule[key] = 1
+                break
+            index += 1
+
```

Comment:
Thank you for your willingness to contribute! This is a great start. I had a few questions about your implementation.

Is this function "find_time()" meant to return anything? Right now it looks like it currently returns nothing, and in order to modify the existing "schedule" dictionary, it would make sense for it to return something.

Additionally, would use of python's built-in "range()" or "enumerate()" functions make more sense here? Or at least be easier to read? Just wondering what your thoughts are as to the functionality and readability of your code changes before I merge the PR.

Thanks again!

-Rela